src/components/AddNewBookForm/AddNewBookForm.js

```diff
@@ -165,10 +165,7 @@ const AddNewBookFormBase = props => {
   return (
     <div className="add-book-form">
       <div className={'upload-progress ' + progressStyle}>
-        <h3>
-          Upload succeeded, redirecting to books overview page. It can take a
-          couple of minutes for your book to be indexed.
-        </h3>
+        <h1>Upload succeeded, redirecting to books overview page.</h1>
```

Comment:
By all means, don't us ```h1``` here, since it is against all SEO practices.
The ```h1``` of the page is the most prominent text when it comes to text search indexing in SERPs such as Google.

Now you are telling Google that the most important text on this page is "Upload succeeded, redirecting to books overview page.".

You have to make it something like ```<h3>``` or ```<p>```and then style it so that it looks like what it looked like before.

Reply:
One Page App doesn't support SEO unless you use Server Side Rendering.

Reply:
Anyway, I can change this to what you prefer.